strr-strata-web/app/components/form/ReviewConfirm.vue

```diff
@@ -312,43 +312,37 @@ onMounted(async () => {
       </FormCommonReviewSection>
     </ConnectPageSection>
 
-    <section class="space-y-6">
-      <h2>{{ $t('word.Certify') }}</h2>
-
-      <UForm
-        ref="strataConfirmationFormRef"
-        :state="applicationStore.confirmation"
-        :schema="applicationStore.confirmationSchema"
-        class="space-y-10 pb-10"
+    <UForm
+      ref="strataConfirmationFormRef"
+      :state="applicationStore.confirmation"
+      :schema="applicationStore.confirmationSchema"
+      class="space-y-10 pb-10"
+    >
+      <ConnectFormCertify
+        v-model="applicationStore.confirmation.confirmation"
+        :title="$t('strr.label.confirmation')"
+        :items="[{ slot: 'item-1' }, { i18nKey: 'certify.2' }]"
+        :checkbox-label="{ key: 'certify.checkbox' }"
```

Comment:
can pass this a regular string or an i18n key with whatever props you want eg: name